wagtail/wagtailforms/models.py

```diff
@@ -295,7 +295,15 @@ def process_form_submission(self, form):
 
     def send_mail(self, form):
         addresses = [x.strip() for x in self.to_address.split(',')]
-        content = '\n'.join([x[1].label + ': ' + text_type(form.data.get(x[0])) for x in form.fields.items()])
+        content = []
+        for field in form:
+            value = field.value()
+            if isinstance(value, list):
+                value = ', '.join(value)
+                if len(value) == 0:
```

Comment:
Well, yes, here I was simply following the original output of this function.  If you look at the tests (https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailforms/tests/test_models.py#L288) then you can see that the default behaviour was to include a string value of None (probably the default output of the get function when no value was present).